src/internal/coredb/projects.go

```diff
@@ -73,6 +111,11 @@ func DeleteProject(ctx context.Context, queryExecer QueryExecer, projectName str
 	return nil
 }
 
+func DeleteAllProjects(ctx context.Context, queryExecer QueryExecer) error {
+	_, err := queryExecer.ExecContext(ctx, "TRUNCATE core.projects CASCADE;")
```

Comment:
I think we should be very careful with CASCADE. In fact, I don't think we want to implement that right now. I think it's safer to delete everything in order in Go code, instead of trusting SQL cascade deletes.

Reply:
It seems like there is some disagreement but I think I would like to do as much of it in postgres as possible only if doing it in postgres makes the code a lot simpler.

Reply:
For now I can omit cascade and we can come back to it later.